doc/source/user_guide/krylov.rst

```diff
@@ -0,0 +1,156 @@
+=======================================
+Harmonic Analysis via the Krylov Method
+=======================================
+
+Introduction
+============
+You can use the frequency-sweep Krylov method for a high performance solution of forced-frequency simulations 
+in acoustic or single-field structural analyses.
+
+Similar to the full harmonic analysis, the frequency sweep Krylov method also uses the full system matrices 
+to compute the harmonic response. While the full method solves the full system matrices at every frequency point 
+in the frequency range, the frequency-sweep Krylov method performs the following steps to approximate the response 
+across the frequency range :
+
+* Builds a Krylov subspace set of vectors at the frequency value in the middle of the requested frequency range,
+* Reduces the system matrices and loading on the entire frequency range,
+* Solves the reduced system, and
+* Expands the results back to compute the harmonic response. 
+
+MAPDL provides the following methods to implement a harmonic analysis using the Krylov Method:
+
+#. Via Mechanical APDL commands.
+#. Via macros, a feature described in *Frequency-Sweep Harmonic Analysis via the Krylov Method* in the *Structural Analysis Guide*.
+#. PyMAPDL implementation of the macros, a feature described here.
```

Comment:
```suggestion
MAPDL provides the following ways to implement a harmonic analysis using the Krylov Method:

#. Via Mechanical APDL commands. You can use APDL macros as described in *Frequency-Sweep Harmonic Analysis via the Krylov Method* in the *Structural Analysis Guide*.
#. PyMAPDL. Use Python programming language to expose the Krylov features as described here.
```

Reply:
Line 24 changed to "#. PyMAPDL. You can use the Python programming language to expose the Krylov features as described in this section."

Reply:
I would like to retain that MAPDL has 2 different methods to do this :
something like this :
"MAPDL provides the following ways to implement a harmonic analysis using the Krylov method:
#. Mechanical APDL commands.
#. APDL macros as described in "Frequency-Sweep Harmonic Analysis via the Krylov Method" in the *Structural Analysis Guide*.
#. PyMAPDL. You can use the Python programming language to expose the Krylov features as described in this section.
"